pwa/src/templates/templateParts/footer/FooterTemplate.tsx

```diff
@@ -1,273 +1,299 @@
+      <div className={styles.container}>
+        <div className={styles.contentGrid}>
+          {footerContent?.map((content, idx) => (
+            <DynamicSection key={idx} {...{ content }} />
+          ))}
         </div>
 
-        <div className={styles.navigation}>
-          <ul className={styles.list}>
-            <Heading4 className={styles.heading}>{t("Conduction")}</Heading4>
-
-            <li>
-              <Link href="tel:+31853036840">
-                <Icon>
-                  <IconPhone />
-                </Icon>
-                Bel Conduction
-              </Link>
-            </li>
-
-            <li>
-              <Link href="mailto:info@conduction.nl">
-                <Icon>
-                  <IconMail />
-                </Icon>
-                Mail Conduction
-              </Link>
-            </li>
-
-            <li>
-              <Link target="_new" href="https://conduction.nl/">
-                <Icon>
-                  <IconExternalLink />
-                </Icon>
-                Bezoek de website
-              </Link>
-            </li>
-          </ul>
-
-          <ul className={styles.list}>
-            <Heading4 className={styles.heading}>{t("Gemeente Rotterdam")}</Heading4>
-
-            <li>
-              <Link href="tel:14010">
-                <Icon>
-                  <IconPhone />
-                </Icon>
-                Bel Gemeente Rotterdam
-              </Link>
-            </li>
-
-            <li>
-              <Link target="_new" href="https://rotterdam.nl/">
-                <Icon>
-                  <IconExternalLink />
-                </Icon>
-                Bezoek de website
-              </Link>
-            </li>
-          </ul>
-
-          <ul className={styles.list}>
-            <Heading4 className={styles.heading}>{t("An initiative of")}</Heading4>
-
-            <div className={styles.logosContainer}>
-              <img onClick={() => window.open("https://www.rotterdam.nl/")} src={rotterdamLogoSource} />
-              <img onClick={() => window.open("https://www.conduction.nl/")} src={conductionLogoSource} />
-            </div>
-          </ul>
+        <div className={styles.logoAndConduction}>
+          <Logo />
+          <WithLoveByConduction />
         </div>
-      </Container>
+      </div>
     </PageFooter>
   );
 };
+
+const DynamicSection: React.FC<{ content: TDynamicContentItem }> = ({ content }) => {
+  const { t } = useTranslation();
+  const [, setFilters] = React.useContext(FiltersContext);
+
+  return (
+    <section>
+      <Heading3 className={styles.dynamicSectionTitle}>{t(content.title)}</Heading3>
+
+      {content.items.map((item, idx) => (
+        <div key={idx} className={styles.dynamicSectionContent}>
+          {item.label && <strong>{t(item.label)}</strong>}
+          {/* External Link */}
+          {item.link && item.link.includes("http") && (
```

Comment:
I suggest making four local components, for the links. Making the dynamic return statement much easier to read.

Resulting in (e.g.)

```
const DynamicSection: React.FC<{ content: TDynamicContentItem }> = ({ content }) => {
  const { t } = useTranslation();
  const [, setFilters] = React.useContext(FiltersContext);

  return (
    <section>
      <Heading3 className={styles.dynamicSectionTitle}>{t(content.title)}</Heading3>

      {content.items.map((item, idx) => (
        <div key={idx} className={styles.dynamicSectionContent}>
          {item.label && <strong>{t(item.label)}</strong>}

          {item.link && item.link.includes("http") && <ExternalLink />}

          {item.link && !item.link.includes("http") && <InternalLink />}

          {item.markdownLink && !item.setFilter && <MarkdownLink />}

          {!item.link && item.setFilter && <FilterLink />}

          {!item.knk && !item.setFilters && !item.markdownLink && <NoLink />}
        </div>
      ))}
    </section>
  );
};
```